src/main/kotlin/racingcar/validator/Validator.kt

```diff
@@ -0,0 +1,25 @@
+package racingcar.validator
+
+import racingcar.Messages
+
+object Validator {
+    private const val MAX_NAME_LENGTH = 5
+
+    fun validateTries(tries: String): Int {
+        require(tries.toIntOrNull() != null) { Messages.NOT_INTEGER }
+
+        val triesInt = tries.toInt()
+        require(triesInt > 0) { Messages.NOT_SMALLER_THAN_ZERO }
+
+        return triesInt
```

Comment:
tries에 대해 toIntOrNull()과 toInt()를 각각 호출하면서 타입 변환을 두 번 일어나고 있는데 아래처럼 작성하면 한번만 변환해도 되어서 조금 더 효율적일 것 같습니다!
```suggestion
        val triesInt = tries.toIntOrNull()
        require(triesInt != null) { Messages.NOT_INTEGER }
        require(triesInt > 0) { Messages.NOT_SMALLER_THAN_ZERO }

        return triesInt
```

Reply:
아 그렇네요... 정확한 예시까지 감사합니다!